### <center> EXP 7  Dropwise and Filmwise Condensation

### **Objective**:
* To study the Dropwise and Filmwise condensation phenomena.

### **Aim**:
* To determine the inside and outside heat transfer coefficient of Filmwise and Dropwise condensers.
* To compare overall heat transfer coefficient of dropwise and filmwise condenser.

### **Theory**:

**Condensation Heat Transfer:**

The process of condensation is the reverse of boiling. Whenever a saturated vapor comes in contact with a surface at a lower temperature, condensation occurs. There are two modes of condensation: 

a) filmwise, in which the condensate wets the surface forming a continuous film, which covers the entire surface, and 

b) dropwise, in which the vapor condenses into small liquid droplets of various sizes, which fall down the surface in a random fashion.

Unless specially treated, most materials are wettable & as condensation occur a film condensate spreads over the surface. In this type of condensation, the film covering the entire surface grows in thickness as it moves down the surface by gravity. Fresh vapor condenses on to the outside of the film & heat is transferred by conduction through the film to the metal surface beneath. There exists a thermal gradient in the film and so it acts as a resistance to heat transfer.

By specially treating the condensing surface, the contact angle can be changed and the surface becomes ‘non-wettable’. By modifying the surface properties using additives and surface coatings, it is possible to achieve dropwise condensation. It is generally difficult to maintain this type of condensation for long time. Condensation that starts as dropwise generally converted into filmwise condensation after some time. Hence, in all practical applications filmwise condensation is assumed. In order to minimize the thermal resistance of the condensing film, length of the surface is kept small.
In dropwise condensation a large portion of the area of the plate is directly exposed to the vapor, making heat transfer rates much larger (5 to10 times) than those in filmwise condensation.
 

### **Procedure** :
**Starting Procedure:**
1. Ensure that ON/OFF switches given on the panel are at OFF position.
2. Close all the drain valves.
3. Open the funnel valve and air vent valve provided at the top of steam generator.
4. Fill water in the steam generator up to 3/4th of its capacity by observing the level of water in the level indicator.
5. Close the funnel valve and air vent valve.
6. Switch On the main supply.
7. Set the temperature of required steam with the help of DTC, say 100 $^{\circ}C$.
8. Switch ON the heater and wait until steam temperature reaches to required value.
9. Ensure that wet steam vent valve and needle valve provided at front are closed.
10. Allow steam to pass through the pipe and slowly open wet steam vent valve to release wet steam from the pipe.
11. Close the vent valve.
12. Connect cooling water supply.
13. Open the valve to allow cooling water to flow through desired condenser (Ensure that during experiment, water is flowing only through the condenser under test
and second valve is closed).
14. Set the flow rate of cooling water by Rota meter.
15. Open the needle valve slowly to allow steam to enter in the test section and start
the stopwatch to measure mass of steam condensed.
16. Observe that the Steam is condensed on the tubes, and falls down in the glass
cylinder (Depending upon type of condenser under test Dropwise or Filmwise
condensation can be visualized.)
17. After steady state, record the temperature, flow rate of cooling water, pressure.
18. Close the needle valve.
19. Stop the Stopwatch and open the drain valve of the glass chamber to measure the
condensate for particular time.
Closing Procedure:
1. When experiment is over, switch OFF the heater.
2. Partially open air vent valve at steam generator to release the steam.
3. Stop cooling water supply to the condenser.
4. Switch OFF main supply.
5. Drain the apparatus completely after getting cool.


### **Experimental Setup** :
1. Entire setup <img src="setup_condens.jpeg" width=600 height=800 />
2. Glass chamber <img src="glass_chamber.jpeg" width=300 height=250 />
3. Pressure gauge <img src="gauge.jpeg" width=300 height=250 />

### **Observations** :
<img src="obs.png" width=600 height=650 />

### **Formulae** :
1. $$
h_{\text {of } f i l m}=0.926 k_{L}\left[\frac{g \rho_{L}^{2}}{\mu_{L} \Gamma_{v}}\right]^{\frac{1}{3}}
$$
<br/>
2. $$\\
R e_{c}=\frac{4 \Gamma_{v}}{\mu_{L}}
$$ 
<br/>
3. $$
\Gamma_{v}=\frac{W_{c}}{N_{t} \pi d_{0}}
$$
<br/>
4. $$
N u_{d}=0.023\left(R e_{d}\right)^{0.8}(P r)^{n} \text { (Dittus -Boelter equation} \text {) }
$$
<br/>
5. $$
N u_{d}=1.86\left(R e_{d}P r\right)^{1/3} \text { (Sieder and Tate equation} \text {) }
$$
<br/>
6. $$
\frac{1}{U_{i}}=\frac{1}{h_{i}}+\frac{d_{i}}{d_{o}} \frac{1}{h_{o}}
$$


In [86]:
import numpy as np

In [74]:
#general data
d0 = 0.019
di = 0.016
L = 0.175
Ai = 0.00879
A0 = 0.010445
#Experimental data
#For dropwise
mw = 40                               #lph
T2 = np.mean([44.8,45.3,46.7])        #Temperature of surface of chrome plated condensor
T3 = np.mean([77.1,77.2,76.6])        #Temperature of steam in the column
T4 = np.array([37])                   #water inlet temp
T5 = np.mean([39,38.8,38.9])          #water outlet temp
vdrop = 102                 #ml
t = 15                      #min
ms = vdrop/t                #ml/min
lambda_s = 2257.2           #kJ/kg
print(T2,T3,T4[0],T5)

45.6 76.96666666666667 37 38.9


In [77]:
rho_s = 973.7632               #linear interpolation
rho_w = 993.25                 #linear interpolation
Cp_w = 4178.5
M_s = (ms*rho_s*1e-6)/60       #in kg/s
M_w = (mw*rho_w*1e-3)/3600     #in kg/s
Q_s = M_s*lambda_s*1e3
Q_w = M_w*Cp_w*(T5-T4)
Qavg = (Q_w + Q_s)/2
print("The average heat transfered Q = {} W".format(Qavg[0]))

#Inside heat transfer coefficient
#data taken from JP Holman tbk 10th edition
Tb = (T4+T5)/2
rho = 993
mu = 6.82e-4
Pr = 4.53
k = 0.63
V = (M_w/rho)*(4/(np.pi*(di**2)))
Re_d = (rho*V*di)/mu

def Nud (Pr,Re_d):
    if (Re_d > 2500):
        return 0.023*(Re_d**(0.8))*(Pr**(0.4))
    elif (Re_d < 2500):
        return 1.86*(Re_d*Pr)**(1/3)
Nu = Nud(Pr,Re_d)
hi = Nu*k/di
print("Reynolds no. : {}".format(Re_d))
print("Nusselt no. : {}".format(Nu))
print("Inside heat transfer coefficient empirical : {} W/m2 C".format(hi))

#hi value from experimental data
delT1 = T3 - T2
delT2 = T5 - T4
#using LMTD
delTM = (delT2 - delT1)/(np.log(delT2/delT1))
print("\u0394 T_m = {}".format(delTM[0]))
hi_exp = Qavg/(Ai*delTM)
print("Inside heat transfer coefficient from  experiment is :{} W/m2 C".format(hi_exp[0]))

#h0 value from experimental data
h0_exp = Qavg/(A0*delTM)
print("Outside heat transfer coefficient from  experiment is :{} W/m2 C".format(h0_exp[0]))

The average heat transfered Q = 168.36077414948883 W
Reynolds no. : 1287.721140648624
Nusselt no. : 33.482634347113446
Inside heat transfer coefficient empirical : 1318.378727417592 W/m2 C
Δ T_m = 10.50920221408782
Inside heat transfer coefficient from  experiment is :1822.5619128049245 W/m2 C
Outside heat transfer coefficient from  experiment is :1533.7787662570881 W/m2 C


In [78]:
#overall heat transfer coefficients
ui_drop = hi_exp*d0*h0_exp/(d0*h0_exp + di*hi_exp)
print("The overall heat transfer coefficient in dropwise is {} W/m2 C" .format(ui_drop[0]))

The overall heat transfer coefficient in dropwise is 910.9809507004771 W/m2 C


In [79]:
#For filmwise
T1 = np.mean([39.4,39.9,39.6])         #Temperature of surface of plain copper condensor
T3 = np.mean([73.8,75.3,73.2])         #Temperature of steam in the column
T4 = np.array([33.1])                  #water inlet temp
T5 = np.array([34.2])                  #water outlet temp
vfilm = 90                  #ml
t = 15                      #min
fl_film = vfilm/t
print(T1,T3,T4,T5)

39.63333333333333 74.10000000000001 [33.1] [34.2]


In [80]:
#properties at T3 = 74 for steam 
#properties of cooling water at T4 = 33.1
ms = 6
mw = 40
rho_s = 975.47                 #linear interpolation
rho_w = 993.25                 #linear interpolation
Cp_w = 4192.9
M_s = (ms*rho_s*1e-6)/60       #in kg/s
M_w = (mw*rho_w*1e-3)/3600     #in kg/s
Q_s = M_s*lambda_s*1e3
Q_w = M_w*Cp_w*(T5-T4)
Qavg = (Q_w + Q_s)/2
print("The average heat transfered Q = {} W".format(Qavg[0]))

#Inside heat transfer coefficient
Tb = (T4+T5)/2
rho = 994.4
mu = 7.43e-4
Pr = 4.96
k = 0.624
V = (M_w/rho)*(4/(np.pi*(di**2)))
Re_d = (rho*V*di)/mu

def Nud (Pr,Re_d):
    if (Re_d > 2500):
        return 0.023*(Re_d**(0.8))*(Pr**(0.4))
    elif (Re_d < 2500):
        return 1.86*(Re_d*Pr)**(1/3)
Nu = Nud(Pr,Re_d)
hi = Nu*k/di
print("Reynolds no. : {}".format(Re_d))
print("Nusselt no. : {}".format(Nu))
print("Inside heat transfer coefficient : {}".format(hi))

#hi value from experimental data
delT1 = T3 - T1
delT2 = T5 - T4
#using LMTD
delTM = (delT2 - delT1)/(np.log(delT2/delT1))
print("\u0394 T_m = {}".format(delTM[0]))
hi_exp = Qavg/(Ai*delTM)
print("Inside heat transfer coefficient from  experiment is :{} W/m2 C".format(hi_exp[0]))

The average heat transfered Q = 135.5418648527778 W
Reynolds no. : 1181.9997549426132
Nusselt no. : 33.538679951598986
Inside heat transfer coefficient : 1308.0085181123604
Δ T_m = 9.68642733507122
Inside heat transfer coefficient from  experiment is :1591.9189650230908 W/m2 C


In [81]:
#outside heat transfer coefficient for filmwise
Tb = (T1+T3)/2
N_t = 1
g = 9.81
rho_l = 998.72
mu_l = 1.158
k_l = 0.583
Pr_l = 8.184
vert_tube_load = M_s/(N_t*np.pi*d0)
Re_l = 4*vert_tube_load/mu_l
ho_l = 0.926*k_l*(g*(rho_l**2)/(mu_l*vert_tube_load))**(1/3)
print("The outside heat transfer cofficient (empirical):{} W/m2 C" .format(ho_l))
#ho from experiment
h0_exp = Qavg/(A0*delTM)
print("The outside heat transfer coefficient from  experiment is :{} W/m2 C".format(h0_exp[0]))

The outside heat transfer cofficient (empirical):933.5237558872556 W/m2 C
The outside heat transfer coefficient from  experiment is :1339.680967214262 W/m2 C


In [84]:
#overall heat transfer coefficients
ui_film = hi_exp*d0*h0_exp/(d0*h0_exp + di*hi_exp)
ui_empr = hi*d0*ho_l/(d0*ho_l + di*hi)
print("The overall heat transfer coefficient for film wise from experiment: {} W/m2 C" .format(ui_film[0]))
print("The overall heat transfer coefficient for film wise from empirical relation: {} W/m2 C" .format(ui_empr))

The overall heat transfer coefficient for film wise from experiment: 795.6974421587596 W/m2 C
The overall heat transfer coefficient for film wise from empirical relation: 600.0266616662199 W/m2 C


### **Results :**

For **Dropwise**,

The overall heat transfer coeffcient is 910.98 $ W/m^{2} s$

For **Filmwise**,

The overall heat transfer coeffcient from empirical formula is 600.026 $ W/m^{2} s$

The overall heat transfer coeffcient from experiment is 795.697 $ W/m^{2} s$

### **Inferences** :

The following conclusions can be made from the experiment:
1. From both the empirical and experimental calculation that dropwise condensation has the highest heat 
 transfer coefficient (by a factor of 1.5) compared to the filmwise.
2. Formation of film creates more resistance thereby reducing the heat transfer coefficient.
3. Ambiguity arises in the experimental and empirical results due to the errors caused in the experiment. 
4. Results may vary from theory to some degree due to unavoidable heat losses from the glass tube walls.
 

### **Recommendations**:
1. The current setup encloses both the condensor within a glass chamber. This design can be improved by providing separate chambers as it will result in the precise measurement of condensate collected.
2. The glass chamber can be constructed using a material in which dropwise condensation is minimized. As in the existing model there is condensate contribution from the glass chamber which in turn affects the reading.
2. Improved insulation for the apparatus.
3. The cooling water supply is to be maintained in a thermally insulated area.

### **Industrial Applications :**

From wearable devices and wireless sensors tracking cargo on ships, to outdoor electronics, more devices are subjected to harsh environments than ever before. Gasketing and sealing enclosures, in addition to being extremely costly, often leave circuitry vulnerable to condensation. One solution is to give barrier coatings to provide a perfect pinhole-free dielectric insulator whose barrier properties helps protect sensitive electronics from water & salt spray, sweat, solvents, sunscreen, vapors, and dust while adding negligible weight & thickness. This utlimately prevents dropwise condensation on its surface. 